packages/core/src/scss/component/accordion/_index.scss

```diff
@@ -448,6 +476,94 @@
+  }
+
+  #{defaults.$accordion-expand-selector} {
+    z-index: 2;
+    margin: $accordion-expand-margin;
+    padding: $accordion-expand-padding;
+    transform: $accordion-expand-transform;
+    transition: $accordion-expand-transition;
+    border-width: $accordion-expand-border-width;
+    border-style: $accordion-expand-border-style;
+    border-radius: $accordion-expand-border-radius;
+
+    @each $selector in map.get(root-defaults.$state-selectors, visited) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-expand-transform: #{$accordion-expand-visited-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, focus) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-expand-transform: #{$accordion-expand-focus-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, hover) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-expand-transform: #{$accordion-expand-hover-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, active) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-expand-transform: #{$accordion-expand-active-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, disabled) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-expand-transform: #{$accordion-expand-disabled-transform};
+      }
+    }
+  }
+
+  #{defaults.$accordion-collapse-selector} {
+    z-index: 2;
+    margin: $accordion-collapse-margin;
+    padding: $accordion-collapse-padding;
+    transform: $accordion-collapse-transform;
+    transition: $accordion-collapse-transition;
+    border-width: $accordion-collapse-border-width;
+    border-style: $accordion-collapse-border-style;
+    border-radius: $accordion-collapse-border-radius;
+
+    @each $selector in map.get(root-defaults.$state-selectors, visited) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-collapse-transform: #{$accordion-collapse-visited-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, focus) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-collapse-transform: #{$accordion-collapse-focus-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, hover) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-collapse-transform: #{$accordion-collapse-hover-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, active) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-collapse-transform: #{$accordion-collapse-active-transform};
+      }
+    }
+
+    @each $selector in map.get(root-defaults.$state-selectors, disabled) {
+      &#{$selector} {
+        --#{root-defaults.$prefix}accordion-collapse-transform: #{$accordion-collapse-disabled-transform};
+      }
+    }
```

Comment:
States have been made a lot easier to handle- you can see how it's done higher up in this file with `@include state.generate-modifiers()`. Make sure you update this.